packages/cozy-harvest-lib/src/models/konnector/KonnectorJobWatcher.js

```diff
@@ -66,6 +66,7 @@ export class KonnectorJobWatcher {
       clearTimeout(this.successTimer)
       this.successTimer = null
     }
+    this.realtime.unsubscribeAll()
```

Comment:
Nit: It is a bit surprsing to see the subscribe inside watch and the unsubscribe inside disableSuccessTimer. It is easier to audit when handlers are added/removed in symmetrically called methods.

Should the realtime subscribe be done inside enableSuccessTimer ?